docs/docs/install-shells.mdx

```diff
@@ -109,5 +110,19 @@ config set prompt  "(oh-my-posh --config ~/.poshthemes/jandedobbeleer.omp.json |
 
 Restart nu shell for the changes to take effect.
 
+</TabItem>
+
+<TabItem value="Cmder/Clink/ConEmu">
```

Comment:
Why `Cmder/Clink/ConEmu` and not CMD?

Reply:
Because there is currently no way to make oh-my-posh work with native CMD AFAIK, we have to use it with something that uses CLink.

Reply:
Help me understand why we can't add that as a prerequisite? Because we're now mixing terminals with shells, that's the issue I see here. As far as I'm aware, only CMD needs a specific clink integration as all other shells have the ability to run oh-my-posh (regardless of the terminal).

Reply:
Excuse me if I'm mixing up the terminology, I am not aware of the difference between a terminal/shell.

I got my information from the discussion in #21, I understood from your comments that it's not possible to replace cmd native shell(?) with oh-my-posh. Is this not true anymore? Can we do that with good old CMD now?

Reply:
I think I misunderstood your last message, if what you mean that the real requirement here is CLink (shell), and ConEmu and Cmder  (terminals?) both use CLink, therefore the tab label should be just CLink, I think we should still keep all 3, because even though it would be technically correct, I anticipate that many users who use Cmder/ConEmu wouldn't know that they use CLink, and mentioning all 3 would make it easier for them to find it.

Reply:
What I meant is that the only shell where this is relevant (or useful) is CMD. For every other shell running inside ConEmu, CMDer that isn't very useful as you can set up oh-my-posh "natively" in the shell itself (Bash, PowerShell, WSL shells, ...). So the actual use-case is "How do I use oh-my-posh for CMD?" which has nothing to do with CMDer/Conemu.

Reply:
What we need is the following:
- Change the label to CMD
- Explain that CMD doesn't have native support for prompts, but you can have the same effect using clink

````md
CMD does not come with native support to change the prompt. You can however make use of clink
to display oh-my-posh.

Integrating oh-my-posh with Clink is easy: just save the following text to an oh-my-posh.lua file in your Clink scripts directory (run clink info to find that), and make sure the oh-my-posh.exe program is in a directory listed in the %PATH% environment variable. (Or edit the script below to provide a fully qualified path to the oh-my-posh.exe program.)

 ```lua
 local custom_prompt = clink.promptfilter(50)
 function custom_prompt:filter(prompt)
     return io.popen("oh-my-posh --config ~\\jandedobbeleer.omp.json"):read("*a")
 end
```

Restart CMD for the changes to take effect.
````